src/relax/transform/remove_purity_checking.cc

```diff
@@ -52,6 +52,12 @@ class PurityRemover : public ExprMutator {
                       call->attrs, call->sinfo_args);
       return VisitExpr(ret);
     }
+    if (call->op == call_inplace_packed_op_) {
+      // call_inplace_packed has its own attrs so we don't pass those down
+      auto ret = Call(call->args[0], Array<Expr>(call->args.begin() + 1, call->args.end()),
+                      tvm::Attrs(), call->sinfo_args);
```

Comment:
As far as I know, we never use the attrs for `call_pure_packed`, so we could simply drop those and combine those cases